hw/ip/otp_ctrl/data/otp_ctrl_mmap.hjson

```diff
@@ -222,22 +222,26 @@
                 {
                     name:        "EN_SRAM_IFETCH",
                     size:        "1",
-                    inv_default: "0",
+                    ismubi:      true,
+                    inv_default: false
```

Comment:
btw, can you remind me why we call it "inv_default"? does this mean the invalid value this field takes on when the partition is invalid due to errors?

Reply:
yes, exactly. this is the default value in invalid state when the partition has not been loaded and validated yet, or if an error has occurred.